interpreter/cling/lib/Interpreter/Interpreter.cpp

```diff
@@ -666,6 +695,62 @@ namespace cling {
   }
 
   Interpreter::CompilationResult
+  Interpreter::codeComplete(const std::string& line, size_t& cursor,
+                            std::vector<std::string>& completions) const {
+
+    const char * const argV = "cling";
+    std::string llvmdir = this->getCI()->getHeaderSearchOpts().ResourceDir;
```

Comment:
Yes, it was just clearer when implementing so not to confuse the parent with the child